### PR TITLE
fix: contact us template and dto body

### PR DESCRIPTION
**Pull request by Amal-Salam**
### What does this Pr Do.
This pr implements the fix for the contact us endpoint of the boiler plate by redirecting the email template to the proper folder, redesigning to be more in suit with the remaining templates. this pr also cleans up the contact code by  using constants thus increasing codebase uniformity and including phone number dto to match frontend and design.
<hr> 

### How is it manually Tested?
         POST/api/v1/contact
   ###  Request Body

```
{
  "name": "John Doe",
  "email": "johndoe@example.com",
  "phone": "123-456-7890",
  "message": "I'd like to know more about your services."
}
```
<hr>

#Checklist of what you did
- [x] Redirected the email template to the correct folder path.
- [x] Redesigned the email template to match other templates.
- [x] Cleaned up the contact code by using constants.
- [x] Added the phone field to the ContactDto.
- [x] Tested the POST request to /api/v1/contact and verified email delivery.
<hr>

#Issue reference

### CODE REVIEW

File: src/modules/contact-us/contact-us.service.ts [hunk 1]
```diff
@@ -20,23 +22,25 @@ export class ContactUsService {
     try {
       await this.sendEmail(createContactDto);
     } catch (error) {
-      throw new InternalServerErrorException('Failed to send email');
+      console.log(error.message);
```
Comment (Cobidex) [inline src/modules/contact-us/contact-us.service.ts:25]: Remove console.log

File: src/modules/contact-us/contact-us.service.ts [hunk 2]
```diff
@@ -20,23 +22,25 @@ export class ContactUsService {
...
+      throw new InternalServerErrorException(SYS_MSG.INQUIRY_NOT_SENT);
```
Comment (Cobidex) [inline src/modules/contact-us/contact-us.service.ts:26]: Don't throw internal server error. Let the global interceptor catch and log the error.

Comment (Amal-Salam) [reply src/modules/contact-us/contact-us.service.ts]: noted.
Thank you for the review.
i have resolved this.

File: src/modules/contact-us/contact-us.service.ts (hunk 1 above)
Comment (Amal-Salam) [reply src/modules/contact-us/contact-us.service.ts]: removed

**Review by Cobidex — APPROVED** [review]

**Pull request merged**